cn.apipost.apipost.json

```diff
@@ -0,0 +1,123 @@
+                "install -D NotoSans-ExtraCondensedBold.ttf /app/share/fonts/"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://noto-website-2.storage.googleapis.com/pkgs/NotoSans-hinted.zip",
+                    "sha256": "02e5f834c526f815f3b9f90f98e2b5b09f17cec107cc751c3b58fc3a2c0b2942"
+                }
+            ]
+        },
+        {
+            "name": "noto-font-cjk-regular",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p /app/share/fonts",
+                "install -D NotoSansCJK-Regular.ttc /app/share/fonts/"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://noto-website-2.storage.googleapis.com/pkgs/NotoSansCJK-Regular.ttc.zip",
+                    "sha256": "32d03afb0b1109fa75a96f89dfc118015283a5343f2216ee4b3757c19992f75d"
+                }
+            ]
+        },
+        {
+            "name": "noto-font-cjk-bold",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p /app/share/fonts",
+                "install -D NotoSansCJK-Bold.ttc /app/share/fonts/"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://noto-website-2.storage.googleapis.com/pkgs/NotoSansCJK-Bold.ttc.zip",
+                    "sha256": "3cd895bce6765903b2eefcd90eea6ed372364e66d5b896ea3a594bd3e737eba0"
+                }
+            ]
+        },
+        {
+            "name": "noto-font-cjk-medium",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p /app/share/fonts",
+                "install -D NotoSansCJK-Medium.ttc /app/share/fonts/"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://noto-website-2.storage.googleapis.com/pkgs/NotoSansCJK-Medium.ttc.zip",
+                    "sha256": "d7590637168e1cff4fd3a15422e483308d7ba2a7120ccaf96ead274c3b40e1b3"
+                }
+            ]
+        },
+        {
+            "name": "apipost",
+            "builddir": true,
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm644 cn.apipost.apipost.metainfo.xml -t $FLATPAK_DEST/share/metainfo",
+                "install -Dm644 cn.apipost.apipost.desktop /app/share/applications/cn.apipost.apipost.desktop",
+                "install -Dm644 cn.apipost.apipost.svg /app/share/icons/hicolor/scalable/apps/cn.apipost.apipost.svg",
+                "mkdir -p /app/dir",
+                "mkdir -p /app/bin",
+                "tar xvf apipost_linux_x64_8.1.4.tar.xz -C /app/dir",
+                "ls /app/dir",
+                "chmod +x /app/dir/apipost_x64_8.1.4/apipost",
```

Comment:
you should `mv` to drop the version from this path, otherwise it will break between updates